### ISSUE TITLE
Hi！ how to draw boundary heat map?

### ISSUE THREAD

**Issue by wqz960** [FunkyKoki/Look_At_Boundary_PyTorch#6]
Great job! But I want to get a boundary heat map by inputing a raw image(cropped) and groundtrue landmarks, but I saw your code, i cannot understand.... can you provide me a script for drawing boundary heat map by inputing the image and landmarks instead of dataset in your `get_gt_heatmap(dataset, gt_coords) `function or tell me more details to change it. Thank you!

**Comment by wqz960** [comment]
Thanks to your codes. I have drawn the gt heat maps. but there is another question, the size of heat maps is (13,64,64), the pixel value is [0,1], but the pixel value of raw image from the opencv imreader is [0,255] , does it need to transform the heat maps to [0,255]?

**Comment by FunkyKoki** [comment]
It is great if it helps you.
About the question you asked, use the matplotlib, it can draw heat map by pixel in range(0, 1).


 
---Original---

**Comment by wqz960** [comment]
@FunkyKoki  another question: for the regressor, the input of the network is the concatenate of raw image and boundary heat maps whose size is (batch, 16,256,2556) or others, I do not what are the feature maps. Thank you!!!

**Comment by FunkyKoki** [comment]
cannot understand your question very well, but if I did not forget, the regressor input is acquired by "input image fusion" as described in the paper


 
---Original---

**Comment by wqz960** [comment]
Hi @FunkyKoki the input fusion image for regressor seems different with the original paper, ` H = I ⊕ (M1 ⊗ I) ⊕ ... ⊕ (MT ⊗ I) ` which is ((13+1)*3, 256, 256) size, but your network is only support gray picture,  which is (14,256,256). Am I mistaken in this respect?

**Comment by FunkyKoki** [comment]
indeed, the author did not follow the formula he gave. you should check the prototxt file he gave, and that is what i have done.


 
---Original---
